lib/github_service/commands/cross_repo_test.rb

```diff
@@ -274,15 +274,15 @@ def update_github_workflow_yaml_content
         raw_yaml = rugged_repo.blob_at(branch_ref.target.oid, ".github/workflows/ci.yaml").content
         content  = YAML.safe_load(raw_yaml)
 
-        content["jobs"] = test_repos.each_with_object({}) do |test_repo, result|
-          result[test_repo] = {
+        content["jobs"] = {
+          "cross-repo" => {
             "uses" => "ManageIQ/manageiq-cross_repo/.github/workflows/manageiq_cross_repo.yaml@master",
             "with" => {
-              "test-repo" => test_repo,
-              "repos"     => repos.join(",")
+              "test-repos" => test_repos.inspect,
+              "repos"      => repos.join(",")
```

Comment:
Not for this PR, but it might be nice to do this one as `repos.inspect` for consistency with test-repos.  I know that it's not used in the same way, but it's weird to have one be an "Array" and the other not.